### DISCUSSION TITLE
Для кого книга ?

### DISCUSSION THREAD

**Discussion by gochaorg** [twirl/The-API-Book#6] [Q&A]
Пробежался глазами по книге, и меня смутило примерно следующее

Не точно определен конечный читатель - точнее если это программист, то какими навыками он должен обладать

  * Писать код на языке 1/2/3/4/5
  * Писать серверный код на языке 2/3/4/5
  * Писать многопоточноный код
  * ....

В первой главе очень мало про это, и не упомниаеться о минимальных скилах, чтоб читать книгу

**Comment by twirl** [comment]
Я думал об этом, но не смог эти самые формальные требования сформулировать. Закрываю в пользу #7
